server/db/index.js

```diff
@@ -84,5 +84,6 @@ const ReservationsUser = sequelize.define('reservations_user', {
 
 ReservationsTable.sync();
```

Comment:
@LittleJennie 
@Sam-I-Am92 

Instead of calling two individual models .sync method, you can call sequelize.sync to sync all tables, if that is easier!